word-guess.rb

```diff
@@ -0,0 +1,97 @@
+
+require 'colorize'
+require 'Faker'
+class Game
+  attr_reader  :name
+ def initialize (name)
+   @name = name
+ end
+  def empty_array?(array)
```

Comment:
These 3 methods (`empty_array?`, `display_underlines` and `input_incorrect?`) are all great examples of encapsulating functionality in a method. Great work!